activate.sh

```diff
@@ -32,6 +32,7 @@ ${_file}"
 				fi
 			fi
 			[ "$(\pwd -P)" = "${_mountpoint}" ] && \break
+      [ "$(\pwd -P)" = "/" ] && \break
```

Comment:
Suggestion: replace the whitespace with tabs to align with other formatting.

```suggestion
			[ "$(\pwd -P)" = "/" ] && \break
```

Reply:
@emorikawa can you please update the PR with the suggested change?